client/blocks/signup-form/index.jsx

```diff
@@ -1322,6 +1331,7 @@ class SignupForm extends Component {
 						onInputBlur={ this.handleBlur }
 						onInputChange={ this.handleChangeEvent }
 						onCreateAccountError={ this.handleCreateAccountError }
+						onCreateAccountSuccess={ this.handleCreateAccountSuccess }
```

Comment:
Hey @jeyip 👋  Should this be `onCreateAccountSuccess={ this.props.handleCreateAccountSuccess }`? 🙂  I don't see `handleCreateAccountSuccess` defined in the signup-form component. 🤔

Reply:
Thanks for the ping @chihsuan! Taking a look

Reply:
Yes you're totally correct, but what's odd is that the feature is still functioning as expected 😅

The good news is that things are still behind a feature flag, and this method is only used by said feature. I'll make the corrections later tonight and return tomorrow do a more thorough investigation

Reply:
https://github.com/Automattic/wp-calypso/pull/92310